Bio/Seq.py

```diff
@@ -2527,6 +2527,55 @@ def toseq(self):
         """
         return Seq("".join(self.data), self.alphabet)
 
+    def join(self, other):
+        """Return a merge of the sequences in other, spaced by the sequence from self.
+
+        Accepts all Seq objects and Strings as objects to be concatinated with the spacer
+
+        >>> spacer = MutableSeq('NNNNN')
+        >>> seqlist = list([Seq("AAA"),Seq("TTT"),Seq("PPP")])
```

Comment:
Spaces after the commas to follow PEP8 style please. Also there is no reason to have ``list(...)`` here.